gryphon/data_service/pollers/orderbook/websocket/bitstamp_orderbook_websocket.py

```diff
@@ -230,7 +230,7 @@ def apply_change_to_orderbook(self, change):
 
         # Re-sort the asks.
         self.orderbook['asks'] = OrderedDict(
-            sorted(self.orderbook['asks'].iteritems(), key=lambda (k, v): float(k)),
+            sorted(self.orderbook['asks'].iteritems(), key=lambda k_v: float(k_v[0])),
```

Comment:
It could be prettier a bit.
```suggestion
            sorted(self.orderbook['asks'].iteritems(), key=lambda k, v: float(k)),
```